site/webpack.config.js

```diff
@@ -92,9 +92,13 @@ module.exports = [
       new CopyPlugin({
         patterns: [{ from: join(__dirname, 'src/assets') }],
       }),
-      new BundleAnalyzerPlugin({
-        openAnalyzer: false,
-      }),
+      ...(process.env.CI !== 'true'
```

Comment:
Assuming this works in Netlify CI?

Reply:
Yep [link](https://docs.netlify.com/configure-builds/environment-variables/#netlify-configuration-variables) and tested on branch deployment just in case